docs/book/installation/sylius_plus_upgrading.rst

```diff
@@ -0,0 +1,39 @@
+.. rst-class:: plus-doc
+
+Upgrading Sylius Plus
+=====================
+
+Sylius regularly releases new versions, usually every two weeks.
+Each release comes with an ``UPGRADE.md`` file, which is meant to help in the upgrading process.
+
+1. **Update the Sylius Plus version constraint by modifying the ``composer.json`` file:**
+
+    .. code-block:: yaml
+
+        {
+            "require": {
+                "sylius/plus": "^0.21"
+            }
+        }
+
+2. **Upgrade dependencies by running a Composer command:**
+
+    .. code-block:: bash
+
+        composer update sylius/plus --with-all-dependencies
+
+    If this does not help, it is a matter of debugging the conflicting versions and working out how your ``composer.json``
+    should look after the upgrade.
+    You can check what version of Sylius is installed by running ``composer show sylius/plus`` command.
+
+3. **Follow the instructions found in the ``UPGRADE.md`` file for a given minor release.**
```

Comment:
And add a link to UPGRADE.md of v0.21?

Reply:
not a fan of this idea, it's an example above with this version.

there is one upgrade file for all versions in this repository afaik :D